src/internal/ppsutil/util.go

```diff
@@ -410,7 +411,7 @@ func WriteJobInfo(pachClient *client.APIClient, pipelineJobInfo *pps.PipelineJob
 	return err
 }
 
-func StatsCommit(commit *pfs.Commit) *pfs.Commit {
+func MetaCommit(commit *pfs.Commit) *pfs.Commit {
```

Comment:
I've been meaning to ask, are we intending to just drop the word "stats" everywhere in favor of "meta"?

Reply:
Yes, it's in the low-priority column of the [global ids project](https://github.com/pachyderm/pachyderm/projects/14).